### PR TITLE
Create dependencies.md

### PR DESCRIPTION
**Pull request by mxdvl**
## How to test

Already present in:
- `consent-management-platform` since [c3c18492c](https://github.com/guardian/consent-management-platform/commit/c3c18492c541a5f95962e76c88479999bd5012a6)

- `commercial-core` since [26815b77](https://github.com/guardian/commercial-core/pull/285/commits/26815b772a461608667839d126c46abdbbc564e7)

## How can we measure success?

Allow shared dependencies to get security fixes, and let the responsibility lie with the consumer applications.

## Have we considered potential risks?

Added known issues/packages which are not respecting semantic versioning.

## Accessibility

- [ ] [Tested with screen reader](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/558398)
- [ ] [Navigable with keyboard](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/39894d)
- [ ] [Colour contrast passed](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/92b913)
- [ ] [The change doesn't use only colour to convey meaning](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/29032f)

### CODE REVIEW

File: dependencies.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+> Given a version number `major`.`minor`.`patch`, increment the:
+>
+> - `major` version when you make incompatible API changes;
+> - `minor` version when you add functionality in a backwards compatible manner;
+> - `patch` version when you make backwards compatible bug fixes.
+>
+> Additional labels for pre-release and build metadata are available as extensions to the `major`.`minor`.`patch` format.
+
+Several tools, such as [Dependabot][] or Snyk, can automatically and regularly
```
Comment (tjmw) [inline dependencies.md:19]: Totally a style thing, but I don't think you need the empty `[]` here.

Ditto the `[Semantic Versioning][semver]` link could become just `[Semantic Versioning]` and then link with `[Semantic Versioning]: https://semver.org/`.

**Review by tjmw — APPROVED** [review]
This looks great! Nice work.

I left one minor comment about a couple of the markdown links to take or leave.

File: dependencies.md (hunk 1 above)
Comment (mxdvl) [reply dependencies.md:19]: It works “magically” in Github, but omitting the `[]` [isn’t in the Markdown spec] [1]. I slighlty prefer the more verbose and explicit and don’t find it impedes readability.

[1]: https://daringfireball.net/projects/markdown/syntax#link

Comment (tjmw) [reply dependencies.md:19]: Ahh TIL, thanks for explaining 👍

**Pull request merged**